services/dummy-test-service/dbmethods.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+type user struct {
+	UserID   int
+	Username string
+	Nickname string
+	City     []city
+}
+
+type city struct {
+	CityID    int
+	Name      string
+	Buildings []building
+}
+
+type building struct {
+	BuildingID int
+	Name       string
+	Level      int
+	Available  bool
+}
+
+type dbClient interface {
+	getUserList() map[string][]user
```

Comment:
In the interface we could return (value, err) in all methods, since there might be errors while connecting to the real data base (not on the mock ofc)